### DISCUSSION TITLE
Pilot profiles?

### DISCUSSION THREAD

**Discussion by tb59427** [iltis42/XCVario#218] [Ideas]
Hi,

am a happy xcvario user. Sharing my ship with a friend and he prefers different audio settings than me. Any chance of implementing pilot profiles, so we can save different preferences for different pilots?

cheers
Torsten

**Comment by iltis42** [comment]
The architecture currently can't support user profiles, and limited resources such as flash storage needed a sophisticated concept for such a thing, e.g. sharing common settings, and having a smaller set of individual settings. Entering names is also a challenge with just one rotary button, so if this came, just number might be a solution.

**Comment by tb59427** [comment]
Numbers would be fine for me...
